x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -138,16 +138,7 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
               isLogstashOutput && (
                 <FormattedMessage
                   id="xpack.fleet.editOutputFlyout.logstashTypeOutputBetaHelpText"
-                  defaultMessage="Logstash output is in BETA, Please help by reporting any bugs. {learnMoreLink}."
-                  values={{
-                    learnMoreLink: (
-                      <EuiLink href={docLinks.links.fleet.guide} external>
-                        {i18n.translate('xpack.fleet.editOutputFlyout.learnMoreLink', {
-                          defaultMessage: 'Learn more',
-                        })}
-                      </EuiLink>
-                    ),
-                  }}
+                  defaultMessage="Logstash output is in beta. Please send feedback to report bugs or suggest improvements."
```

Comment:
I've removed the link here because it was a little confusing to point to the guide with no context. I did not add a link to the discuss forum because the page already has a link.

Reply:
How about something like this to indicate there is a Send feedback button.  Also, should it be "and" instead of "or"?

> Click **Send feedback** to report bugs and suggest improvements.